README.md

```diff
@@ -1,11 +1,7 @@
 Balena Documentation
 ======================
 
-[![Build status](https://ci.appveyor.com/api/projects/status/qbsivehgnq0vyrrb?svg=true)](https://ci.appveyor.com/project/fedealconada/resin-docs)
-
-Join our [forums](https://forums.balena.io/), or online chat at [![Gitter chat](https://badges.gitter.im/resin-io/chat.png)](https://gitter.im/resin-io/chat)
-
-Documentation for the [balena](https://balena.io/) platform.
+Documentation for the [balenaCloud](https://balena.io/) platform. Join our [forums](https://forums.balena.io/) to chat.
```

Comment:
Linking to https://www.balena.io definitely makes sense here. We don't prefer https://www.balena.io/cloud here, right?

Reply:
I think either is OK in this context, so LGTM